ldk-server/src/main.rs

```diff
@@ -97,22 +111,62 @@ fn main() {
 								"CHANNEL_PENDING: {} from counterparty {}",
 								channel_id, counterparty_node_id
 							);
+							event_node.event_handled();
 						},
 						Event::ChannelReady { channel_id, counterparty_node_id, .. } => {
 							println!(
 								"CHANNEL_READY: {} from counterparty {:?}",
 								channel_id, counterparty_node_id
 							);
+							event_node.event_handled();
 						},
 						Event::PaymentReceived { payment_id, payment_hash, amount_msat } => {
 							println!(
 								"PAYMENT_RECEIVED: with id {:?}, hash {}, amount_msat {}",
 								payment_id, payment_hash, amount_msat
 							);
+							event_node.event_handled();
+						},
+						payment_forwarded_event @ Event::PaymentForwarded {prev_channel_id,
+							next_channel_id,
+							prev_user_channel_id,
+							next_user_channel_id,
+							total_fee_earned_msat,
+							skimmed_fee_msat,
+							claim_from_onchain_tx,
+							outbound_amount_forwarded_msat} => {
+
+							println!("PAYMENT_FORWARDED: with outbound_amount_forwarded_msat {}, total_fee_earned_msat: {}, inbound channel: {}, outbound channel: {}",
+								outbound_amount_forwarded_msat.unwrap_or(0), total_fee_earned_msat.unwrap_or(0), prev_channel_id, next_channel_id
+							);
+
+							let forwarded_payment = forwarded_payment_to_proto(payment_forwarded_event);
```

Comment:
Do we need to use the proto version here? Seems it would better not to to the translation and allocate a vec.